README.md

```diff
@@ -4,22 +4,26 @@ TimescaleDB metrics reporter for telemetry events emitted by [Membrane Core](htt
 
 Reporter attaches itself to [Telemetry package](https://hex.pm/packages/telemetry) and listens for events declared and documented in Membrane Core's module `Membrane.Telemetry`.
 
+ ### TODO
+ - upgrade TimescaleDB to 2.*,
+ - fix config so that users don't have to configure `ecto_repos`
+ - update readme after integrating changes from `membrane_core/init-and-temrinate-events`
+
 ## Requirements
  - PostgreSQL server instance compatible with TimescaleDB extension.
  
- ** IMPORTANT **
- This reporter is only compatible with TimescaleDB version < 2.0. Make sure to use latest 1.* version.
- 
- ### TODO
- Upgrade TimescaleDB to 2.*.
+ **IMPORTANT:** This reporter is only compatible with TimescaleDB version < 2.0. Make sure to use latest 1.* version.
 
 ## Installation
 
 To make use of the reporter you should add it as a dependency in your application along with `membrane_core`.
 
+**IMPORTANT:** As for now, for the dashboard to work properly it is required to use `init-and-temrinate-events` branch of `membrane_core`.
```

Comment:
To be honest that infomation should probably be be placed in dashboard repository, not here. Reporter supports events that are added in the core branch but it will work nevertheless without it.

Reply:
Fixed in a88f732